desktop/plugins/network/ManageMockResponsePanel.tsx

```diff
@@ -195,7 +199,7 @@ export function ManageMockResponsePanel(props: Props) {
     props.routes,
   ]);
   return (
-    <Container>
+    <Container style={{height: 580}}>
```

Comment:
Note that we now expose `<Layout.Left`, so I suspect the locally styled components aren't largely not needed anymore if you do something like:

```
<Layout.Container height={580}> 
    <Layout.Left>
       <Layout.Container width={200}>
          { /* Left content */ }
        </Layout.Container>
        { /*Right content, probably inside container as well */ }
      </Layout.Left>
 </Layout.Container>
 ```


But if you want to keep the changes small, this is fine as well. Note that you can play with the new layout abstractions by using `View > Flipper Design System` to see some examples.

Reply:
@mweststrate   I would like to keep this change small for now.  Once this is approved I'm planning on putting in the PR for import/export of routes.  Then I'd like to revisit the route UI and add a checkbox to routes for enable/disable.